### PR TITLE
Maps : Add Angouleme campus (fix forgotten entry in others files)

### PR DESCRIPTION
**Pull request by Haletran**

### CODE REVIEW

File: config.sample.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 # Configuration
 redirect_url = "http://{current_domain}/auth"
-auth_link = f"https://api.intra.42.fr/oauth/authorize?client_id={key}&redirect_uri={redirect_url}&response_type=code&scope=public"
+auth_link = "https://api.intra.42.fr/oauth/authorize?client_id={key}&redirect_uri={redirect_url}&response_type=code&scope=public"
```
Comment (wow0000) [inline config.sample.py:11]: Breaking of the key/redirect

**Comment by wow0000** [comment]
Hello !
You are modifying config.py and breaking the key system, please add a commit to your PR to fix this 👍 
You can add commits to your PR without creating a new one 👍

**Comment by Haletran** [comment]
Hello ! Sorry for all those mistakes, it should be fixed now :)

**Comment by wow0000** [comment]
Hello ! I think you messed up you branches :p
https://github.com/wow0000/friends42/pull/14/files
You can check here if the problem is solved
PS: don't worry about learning!!

**Pull request closed without merging**